paasta_tools/check_spark_jobs.py

```diff
@@ -173,13 +208,18 @@ def report_spark_jobs(min_hours, no_notify):
                 update_check_status(service, message, pysensu_yelp.Status.WARNING)
             else:
                 update_check_status(service, 'No long running spark jobs', pysensu_yelp.Status.OK)
+        if email_domain:
+            for result in results:
+                email_user(result, email_domain)
+
```

Comment:
I think it would be cool if there was an `else` here that would print who it would email and what services it would alert

Reply:
The guessed service is already part of messages_by_service we print above. Added a print statement to `email_user`, too.